docs/content/preview/integrations/jaeger.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Jaeger
+linkTitle: Jaeger
+description: Use Jaeger with YCQL API
+menu:
+  preview_integrations:
+    identifier: jaeger
+    parent: integrations
+    weight: 571
+type: docs
+---
+
+Jaeger, inspired by [Dapper](https://research.google.com/pubs/pub36356.html) and [OpenZipkin](http://zipkin.io/), is a distributed tracing system released as open source by [Uber Technologies](http://uber.github.io/). It is used for monitoring and troubleshooting microservices-based distributed systems, including:
+
+- Distributed context propagation
+- Distributed transaction monitoring
+- Root cause analysis
+- Service dependency analysis
+- Performance / latency optimization
+
+Jaeger can be used with various storage backends. YugabyteDB's YCQL API can also be used as a storage backend for Jaeger (version 1.43.0, previous versions are unsupported).
```

Comment:
Can we add link to https://docs.yugabyte.com/preview/api/ycql/ where we mention YCQL.